examples/parent_child_checkbox/lib/main.dart

```diff
@@ -0,0 +1,48 @@
+import 'dart:developer';
```

Comment:
There should be some sample code in the API docs for the widget that shows how to use it (look for `{@tool dartpad` sections in our existing code base for examples). With that, this dedicated example project in the example directory can be removed.